classes/class-wc-rest-connect-shipping-label-print-controller.php

```diff
@@ -61,7 +61,7 @@ public function get( $request ) {
 			);
 		} else {
 			header( 'content-type: ' . $raw_response['headers']['content-type'] );
-			echo $raw_response['body'];
+			echo $raw_response['body']; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
An alternative to ignoring could be allowing restricted HTML, eg: `echo wp_kses_post( $raw_response['body'] );`

This will allow most HTML tags (all HTML tags allowed in `post` contents), but disallows risky ones such as `<script>`.